pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -409,22 +409,31 @@ class DisksDialog(GUIObject):
     mainWidgetName = "disks_dialog"
     uiFile = "spokes/lib/custom_storage_helpers.glade"
 
-    def __init__(self, data, device_tree, disks, selected_disks):
+    def __init__(self, data, device_tree, disks, selected_disks, is_md):
         super().__init__(data)
         self._device_tree = device_tree
         self._selected_disks = selected_disks
         self._disks = disks
+        self.is_md = is_md
+        self.instruction_label = self.builder.get_object("instruction_label2")
         self._store = self.builder.get_object("disk_store")
         self._view = self.builder.get_object("disk_view")
         self._populate_disks()
         self._select_disks()
         self._view.set_tooltip_column(0)
+        self._set_instruction_label()
 
     @property
     def selected_disks(self):
         """Selected disks."""
         return self._selected_disks
 
+    def _set_instruction_label(self):
+        if not self.is_md:
+            self.instruction_label.set_text(_("The device will be installed to "
+                                              "one of the selected drives."))
```

Comment:
I propose to change "The device will be installed to..." to "The mount point will be created on..." - to be more consistent with terms already used in the UI.

Reply:
Hi! I'm a content designer who was asked to take a look! For the content, what do you think of making it a little more concise? Here are a couple of ideas, I'd would love to hear any feedback!
-  "Select all of the drives you would like the mount point to be created on." - wanted to make it user-focused by adding the "you" 
-  "Select 1 or more drives for the mount point to be created on."  - combined the sentences already created to make it a little easier to read and changed "one" to "1" for scalability and consistency (PatternFly recommends using numerals)

Reply:
Great to have you here @wolfeallison , thanks!

Reply:
Hi @wolfeallison sorry for getting back to you this late.
Thank you very much for your advice! I am going to try to explain the situation a little better. Basically we have two use cases for this dialogue. 
- In the first, all of the selected items will be used for the mount point creation. For this I like what you have suggested: *"Select all of the drives you would like the mount point to be created on."*
- In the second, we need to emphasise that even if the user selects multiple items **only one** of them will end up being chosen to have the mount point installed. I think *"Select 1 or more drives for the mount point to be created on."* does not highlight it enough.

Reply:
@OndrejZobal Oh thank you for the explanations! 

For the second situation, what do you think of "Select a drive for the mount point to be created on. If you select multiple, only 1 drive will be used."

Reply:
@wolfeallison I agree this wording is much better. Thank you Allison!